tensorflow/examples/label_image/label_image.py

```diff
@@ -40,33 +40,37 @@ def read_tensor_from_image_file(file_name,
                                 input_width=299,
                                 input_mean=0,
                                 input_std=255):
+  graph = tf.Graph()
+  graph_def = tf.compat.v1.GraphDef()
+
   input_name = "file_reader"
   output_name = "normalized"
-  file_reader = tf.read_file(file_name, input_name)
-  if file_name.endswith(".png"):
-    image_reader = tf.image.decode_png(
+
+  with graph.as_default():
+    file_reader = tf.io.read_file(file_name, input_name)
+    if file_name.endswith(".png"):
+      image_reader = tf.image.decode_png(
         file_reader, channels=3, name="png_reader")
-  elif file_name.endswith(".gif"):
-    image_reader = tf.squeeze(
+    elif file_name.endswith(".gif"):
+      image_reader = tf.squeeze(
         tf.image.decode_gif(file_reader, name="gif_reader"))
-  elif file_name.endswith(".bmp"):
-    image_reader = tf.image.decode_bmp(file_reader, name="bmp_reader")
-  else:
-    image_reader = tf.image.decode_jpeg(
+    elif file_name.endswith(".bmp"):
+      image_reader = tf.image.decode_bmp(file_reader, name="bmp_reader")
+    else:
+      image_reader = tf.image.decode_jpeg(
         file_reader, channels=3, name="jpeg_reader")
-  float_caster = tf.cast(image_reader, tf.float32)
-  dims_expander = tf.expand_dims(float_caster, 0)
-  resized = tf.image.resize_bilinear(dims_expander, [input_height, input_width])
-  normalized = tf.divide(tf.subtract(resized, [input_mean]), [input_std])
-  sess = tf.compat.v1.Session()
-  result = sess.run(normalized)
+    float_caster = tf.cast(image_reader, tf.float32)
+    dims_expander = tf.expand_dims(float_caster, 0)
+    resized = tf.image.resize(dims_expander, [input_height, input_width], method=tf.image.ResizeMethod.BILINEAR)
```

Comment:
style nit: Respect the 80-char line width limit.

Reply:
oops. thanks